lib/rubocop/cop/lint/circular_argument_reference.rb

```diff
@@ -18,11 +18,13 @@ def on_kwoptarg(node)
           arg_name, arg_value = *node
           case arg_value.type
           when :send
-            # ruby 2.0
-            _, name = *arg_value
-            return unless name == arg_name
+            # Ruby 2.0 will have type send every time, and "send nil" if it is
+            # calling a method of its own class with a "self"
+            called_class, name = *arg_value
```

Comment:
Strictly speaking this is a `receiver` object - it might be a Class object, but it might be something else entirely.